sys/dev/isp/ispvar.h

```diff
@@ -551,11 +598,12 @@ struct ispsoftc {
 #define	ISP_CFG_LPORT_ONLY	0x02	/* insist on {N/F}L-Port connection */
 #define	ISP_CFG_NPORT_ONLY	0x04	/* insist on {N/F}-Port connection */
 #define	ISP_CFG_LPORT		0x06	/* prefer {N/F}L-Port connection */
+					/* 0x06 seems wrong to me */
```

Comment:
Its not wrong, since reads are masked by ISP_CFG_PORT_PREF.  It is one 3-bit fields, not three 1-bit fields.